backend/api/Areas/Tools/Helpers/ImportProjectsHelper.cs

```diff
@@ -187,7 +283,8 @@ private Entity.Project Merge(Entity.Project project, Model.ImportProjectModel mo
                     project.AddTask(tasks.DistinctBy(t => t.Id).ToArray());
                     project.Tasks.ForEach(t =>
                     {
-                        t.IsCompleted = !t.Task.IsOptional;
+                        t.IsCompleted = !t.Task.IsOptional; // Complete all required steps.
+                        t.Task = null;
```

Comment:
curios why this is necessary.

Reply:
This question came up today with Maggie too.  It's one of those scenarios where I don't think there is a good solution.  If I let them remain false it will almost certainly block future saves because we validate whether all required tasks are completed.  Which means I'd have to allow for overriding that behaviour...  This gets rather complicated.